### DISCUSSION TITLE
Ventura 13.3.1

### DISCUSSION THREAD

**Discussion by Kitround** [MonitorControl/MonitorControl#1383] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hello,

Seems that the new update of Ventura 13.3.1 is messing with MonitorControl and BetterDisplay

When disconnecting/connecting the external display (LG Ultrafine 5K), the screen is going either full dark, or brighter than before.

I've tried to delete and reinstalling both softwares, same thing.

### Steps to reproduce

/

### Expected behavior

Same brightness

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.3.1
- Mac model: M2
- MonitorControl version: Latest
- Monitor(s): LG Ultrafine
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
There should be no issue like this. Does this happen without the apps running? What if you disable `Software-based video adjustments` under Settings/Displays in BetterDisplay or disable `Combine hardware and software dimming` in MonitorControl?

**Comment by Kitround** [comment]
When i switch to Software-based video adjustments it's coming back to normal until a replug the screen and it's going full dark again. Everything was working well until this update.

**Comment by mhillebrand** [comment]
Since upgrading to macOS 13.3.1, I can no longer control my external LG Ultra HD display from Monitor Control—no matter what I try.

UPDATE: I noticed that it's just my external PC keyboard's brightness keys that are misbehaving. The brightness buttons on my MacBook Pro are still working well with MonitorControl. When I reboot, the problem goes away. It only seems to rear its ugly head once my laptop wakes up from sleep.

**Comment by marcobrambilla** [comment]
same as mhillebrand, except the problem stays even after reboot

**Comment by mhillebrand** [comment]
@marcobrambilla I solved my problem by assigning a custom keyboard shortcut in Preferences: Cmd-F14 and Cmd-F15 now control brightness for the current monitor.

**Comment by waydabber** [comment]
This does not seem to be a general issue, keyboard control should work in 13.3.1, maybe accessibility permissions need a reset if something went wrong during the upgrade. I'll move this to discussions and see if others chime in.
